tests/integration/Owncloud/OcisPhpSdk/ResourceShareLinkTest.php

```diff
@@ -92,8 +93,8 @@ public function testCreateLink(
 
     public function testCreateLinkWithExpiry(): void
     {
-        $tomorrow = new \DateTimeImmutable('tomorrow');
-        $link = $this->fileToShare->createSharingLink(SharingLinkType::VIEW, $tomorrow);
+        $tomorrow = (new \DateTimeImmutable('tomorrow'))->modify('+1 day -1 second');
+        $link = $this->fileToShare->createSharingLink(SharingLinkType::VIEW, $tomorrow, self::PASSWORD);
```

Comment:
I would rather keep on sending only `tomorrow`, because that is what the user can do, but make the assertion check the new expected value